source/documentation/getting_started/quick_start.md

```diff
@@ -1,10 +1,37 @@
 ## Quick start guide
 
-Registers in beta are reliable and are ready for use in products and services.
+Registers which are in beta are reliable and ready for use in products and services. Each register has an open, RESTful API which you can use to access the data in the register. 
 
-1. [Find the base URL for the register(s) you want to use](https://registers.cloudapps.digital/registers). For example, for the `local-authority-eng` register: `https://local-authority-eng.register.gov.uk/`.
-2. [Find the endpoints you want](#apireference). For example, `/record/BIR` in `https://local-authority-eng.register.gov.uk/record/BIR`.
-3. Add `.json`, `.yaml`, `.csv`, `.tsv`, or `.ttl` to the end of the request URL (before any query parameters) to choose the format of the response, or amend headers to `accept application/json` or equivalent.
-4. Parse register data and use it in your product or service.
+You can follow this guide to quickly get started with GOV.UK Registers APIs. 
+
+**1. Find the base URL for the register(s) you want to use.** 
+
+You can find a full list of register base URLs on our [Registers collection page] (https://registers.cloudapps.digital/registers), but the following are some examples:
+
+| Register name | Base URL |
+|----------|----------|
+| `local-authority-eng`     | `https://local-authority-eng.register.gov.uk/`|
+| `country` | `https://country.register.gov.uk/` |
+| `allergen`  | `https://allergen.register.gov.uk/` |
+
+**2. Find the endpoints you want to use.**
+
+For example, `GET record/BIR` is the endpoint for Birmingham City Council in the `local-authority-eng` register.
+
+| Endpoint | Description |
+|----------|----------|
+| [`GET /register`](#get-register)     | View information about a register. |
+| [`GET /records`](#get-records) | Get all records from a register. |
+| [`GET /record/{field-value}`](#get-record-field-value)  | Find a specific record within a register. |
+| [`GET /record/{field-value}/entries`](#get-record-field-value-entries)  | Get all entries for a single record. |
+| [`GET /records/{field-name}/{field-value}`](#get-records-field-name-field-value) | Find all records that share a field-value for a particular field. |
```

Comment:
Probably need to find a way to avoid saying "that share a field value" so @danmitchell-'s idea of renaming it "key" could be useful here.

Reply:
Yes, a bit careless of me re. "key", sorry. Latest commit for this specific PR addresses the backticks around "field-value".